packages/react-app/src/App.jsx

```diff
@@ -226,26 +240,16 @@ function App(props) {
 
       <Switch>
         <Route exact path="/">
-          {/* pass in any web3 props to this Home component. For example, yourLocalBalance */}
-          <Home
-            yourLocalBalance={yourLocalBalance}
-            writeContracts={writeContracts}
-            readContracts={readContracts}
-            address={address}
-            injectedProvider={injectedProvider}
-            userSigner={userSigner}
-          />
+          <Verify {...routeProps} />
         </Route>
-        <Route exact path="/polygons">
-          {/* pass in any web3 props to this Home component. For example, yourLocalBalance */}
-          <Polygons
-            yourLocalBalance={yourLocalBalance}
-            writeContracts={writeContracts}
-            readContracts={readContracts}
-            address={address}
-            injectedProvider={injectedProvider}
-            userSigner={userSigner}
-          />
+        <Route exact path="/create">
+          <CreateZone {...routeProps} />
+        </Route>
+        <Route exact path="/ethdenver">
```

Comment:
What's `/ethdenver` for? Why does it just have the same stuff as Home?

Reply:
"/ethdenver" has the map we released for the hackathon.